### PR TITLE
Updates on lifetime dependence

### PR DESCRIPTION
**Pull request by meg-gupta**

### CODE REVIEW

**Comment by meg-gupta** [comment]
@swift-ci smoke test

**Comment by meg-gupta** [comment]
@swift-ci test

File: include/swift/AST/DiagnosticsSema.def [hunk 1]
```diff
@@ -7871,12 +7871,11 @@ ERROR(lifetime_dependence_only_on_function_method_init_result, none,
...
-      "cannot infer lifetime dependence, multiple ~Escapable or ~Copyable "
-      "parameters with ownership modifiers, specify explicit lifetime "
-       "dependence", ())
-ERROR(lifetime_dependence_cannot_infer_no_candidates, none,
-       "cannot infer lifetime dependence, no parameters with ownership "
-       "modifiers present", ())
+      "cannot infer lifetime dependence, multiple parameters qualifiy as a candidate", ())
+ ERROR(lifetime_dependence_cannot_infer_no_candidates, none,
+       "cannot infer lifetime dependence, no parameters found that are "
+       "~Escapable or Escapable with a borrowing ownership",
```
Comment (atrick) [inline include/swift/AST/DiagnosticsSema.def]: This is still ambiguous. Maybe

`no parameters found that are either ~Escapable or have borrowing ownership.`

**Review by atrick — APPROVED** [review]
One suggestion...

File: include/swift/AST/DiagnosticsSema.def (hunk 1 above)
Comment (meg-gupta) [reply include/swift/AST/DiagnosticsSema.def]: I'll do this as a followup

**Pull request merged**